### ISSUE TITLE
HAN RX Errors starting with 2.2.13

### ISSUE THREAD

**Issue by njordan77** [UtilitechAS/amsreader-firmware#522]
Hi, i started a few months back and did take all updates. with the update v2.2.12->2.2.13 i now see a lot of HAN RX errors and the HAN symbol in RED. not sure if this is a warning or more an issue that communication is broken.

Never seen this before. If its not a generic problem, any idea how i can fix it.

I'm on NodeMCU ESP8266 with TSS721A-Board. The Network is Netz NÖ in Austria with a SAGEMCOM T210-D meter.

Kind Regards
Norbert

**Comment by ArnieO** [comment]
@gskjold : I saw the message today during device testing, before the testjig started sending payloads. The device was in Autodiscovery mode.

**Comment by njordan77** [comment]
Thanks for your replies. I already played arround yesterday with the serial settings. I never understood and found the spec for the connection but realized after some struggle that 8N1 worked til 2.2.12 perfectly. as there was no message.
I changed yesterday to 8E1 and suddenly no more errors showed up. i was not sure if this is the solution and the reason for the errors. but based on your feedback it seems so. MANY Thanks
